README.md

```diff
@@ -140,101 +194,101 @@ string src = "https://ik.imagekit.io/your_imagekit_id/endpoint/default-image.jpg
-| border | b |
-| rotation | rt |
-| blur | bl |
-| named | n |
-| overlayX | ox |
-| overlayY | oy |
-| overlayFocus | ofo |
-| overlayHeight | oh |
-| overlayWidth | ow |
-| overlayImage | oi |
-| overlayImageTrim | oit |
-| overlayImageAspectRatio | oiar |
-| overlayImageBackground | oibg |
-| overlayImageBorder | oib |
-| overlayImageDPR | oidpr |
-| overlayImageQuality | oiq |
-| overlayImageCropping | oic |
-| overlayImageTrim | oit |
-| overlayText | ot |
-| overlayTextFontSize | ots |
-| overlayTextFontFamily | otf |
-| overlayTextColor | otc |
-| overlayTextTransparency | oa |
-| overlayAlpha | oa |
-| overlayTextTypography | ott |
-| overlayBackground | obg |
-| overlayTextEncoded | ote |
-| overlayTextWidth | otw |
-| overlayTextBackground | otbg |
-| overlayTextPadding | otp |
-| overlayTextInnerAlignment | otia |
-| overlayRadius | or |
-| progressive | pr |
-| lossless | lo |
-| trim | t |
-| metadata | md |
-| colorProfile | cp |
-| defaultImage | di |
-| dpr | dpr |
-| effectSharpen | e-sharpen |
-| effectUSM | e-usm |
-| effectContrast | e-contrast |
-| effectGray | e-grayscale |
-| original | orig |
-
-
+The complete list of transformations supported and their usage in ImageKit can be found [here](https://docs.imagekit.io/features/image-transformations). The SDK gives a name to each transformation parameter, making the code simpler and more readable. If a transformation is supported in ImageKit, but a name for it cannot be found in the table below, then use the transformation code from ImageKit docs as the name when using the `url` function.
+
+| Supported Transformation Name | Translates to parameters |
+| --- | --- |
+| Height | h |
+| Width | w |
+| AspectRatio | ar |
+| Quality | q |
+| Crop | c |
+| CropMode | cm |
+| X | x |
+| Y | y |
+| Focus | fo |
+| Format | f |
+| Radius | r |
+| Background | bg |
+| Border | b |
+| Rotation | rt |
+| Blur | bl |
+| Named | n |
+| OverlayX | ox |
+| OverlayY | oy |
+| OverlayFocus | ofo |
+| OverlayHeight | oh |
+| OverlayWidth | ow |
+| OverlayImage | oi |
+| OverlayImageTrim | oit |
+| OverlayImageAspectRatio | oiar |
+| OverlayImageBackground | oibg |
+| OverlayImageBorder | oib |
+| OverlayImageDPR | oidpr |
+| OverlayImageQuality | oiq |
+| OverlayImageCropping | oic |
+| OverlayImageTrim | oit |
+| OverlayText | ot |
+| OverlayTextFontSize | ots |
+| OverlayTextFontFamily | otf |
+| OverlayTextColor | otc |
+| OverlayTextTransparency | oa |
+| OverlayAlpha | oa |
+| OverlayTextTypography | ott |
+| OverlayBackground | obg |
+| OverlayTextEncoded | ote |
+| OverlayTextWidth | otw |
+| OverlayTextBackground | otbg |
+| OverlayTextPadding | otp |
+| OverlayTextInnerAlignment | otia |
+| OverlayRadius | or |
+| Progressive | pr |
+| Lossless | lo |
+| Trim | t |
+| Metadata | md |
+| ColorProfile | cp |
+| DefaultImage | di |
+| Dpr | dpr |
+| EffectSharpen | e-sharpen |
+| EffectUSM | e-usm |
+| EffectContrast | e-contrast |
+| EffectGray | e-grayscale |
+| Original | orig |
```

Comment:
`Raw` method is missing. Also, please use `Raw` instead of `RawTransformation` which is implemented at the moment.